src/xenia/ui/imgui_drawer.cc

```diff
@@ -218,6 +196,130 @@ std::optional<ImGuiKey> ImGuiDrawer::VirtualKeyToImGuiKey(VirtualKey vkey) {
   }
 }
 
+static const ImWchar font_glyph_ranges[] = {
+    0x0020, 0x00FF,  // Basic Latin + Latin Supplement
+    0x0370, 0x03FF,  // Greek
+    0x0400, 0x044F,  // Cyrillic
+    0x2000, 0x206F,  // General Punctuation
+    0,
+};
+
+const std::filesystem::path ImGuiDrawer::GetWindowsFont(std::string font_name) {
```

Comment:
I think this might have been more platform-independent as other operating systems also have their own font storages, and possibly accepting the name (family, weight, italic, etc. possibly) of the font with the necessary lookups in the OS rather than directly the file name. The fallback behavior is also pretty non-obvious in this function. On Windows, it returns an empty path if it has failed to obtain the system fonts directory. However, on other platforms, it just silently tries to get the path from the working directory instead.